api/src/main/java/com/velocitypowered/api/util/title/Title.java

```diff
@@ -0,0 +1,51 @@
+package com.velocitypowered.api.util.title;
+
+/**
+ * Represents a title that can be sent to a Minecraft client.
+ */
+public interface Title {
+    /**
+     * A title that, when sent to the client, will cause all title data to be reset and any existing title to be hidden.
+     */
+    Title RESET = new Title() {
+        @Override
+        public String toString() {
+            return "reset title";
+        }
+    };
+
+    /**
+     * A title that, when sent to the client, will cause any existing title to be hidden. The title may be restored by
+     * a {@link TextTitle} with no title or subtitle (only a time).
+     */
+    Title HIDE = new Title() {
+        @Override
+        public String toString() {
+            return "hide title";
```

Comment:
Is there a reason to have these other than debugging?

Reply:
You would be correct: they're only there for debugging and consistency (since `TextTitle` has `toString()` as well).